bin/thin_client_jaas.conf

```diff
@@ -0,0 +1,5 @@
+ThinClient {
+  com.sun.security.auth.module.Krb5LoginModule required
```

Comment:
I _think_ this will break on IBM java. They do something annoying like have Krb5LoginModule in a different package, I think `com.ibm.security.auth.module.Krb5LoginModule`.

Also annoying, the semantics on what you options you may/must provide in the configuration block are different between vendors.

Reply:
I think these options are OK...

https://www.ibm.com/support/knowledgecenter/SSYKE2_7.1.0/com.ibm.java.security.api.71.doc/jgss/com/ibm/security/auth/module/Krb5LoginModule.html seems to indicate that it's actually `useKeytab` on IBM.

Probably easiest to just acknowledge that we only expect this to work on openjdk/oraclejdk (and maybe azul?). All of those have the exact same JAAS config semantics for krb5, AFAIK.

Reply:
You are right.
I cobbled together a solution without external config file, and with IBM JRE support based on the hadoop minicluster config. It mostly works with IBM JRE now.
Getting the IBM JRE actually talk to a HTTPS server was also fun. (Only old-timers remember the export-only limited cipher brouhaha, that IBM still has)